packages/salesforcedx-vscode-core/test/vscode-integration/util/inputUtils.test.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import { createSandbox, SinonSandbox } from 'sinon';
+import * as vscode from 'vscode';
+import { getTrimmedString } from '../../../src/util/inputUtils';
+
+describe('getTrimmedString', () => {
+  let sandbox: SinonSandbox;
+  const INPUT_VAL = 'Test Input';
+
+  beforeEach(() => {
+    sandbox = createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('should call showInputBox once', async () => {
+    const inputBox = sandbox.stub(vscode.window, 'showInputBox').resolves(INPUT_VAL);
+    const trimmedString = await getTrimmedString({});
+    sandbox.assert.calledOnce(inputBox);
+  });
+
+  it('should remove leading whitespace', async () => {
+    sandbox.stub(vscode.window, 'showInputBox').resolves(`  ${INPUT_VAL}`);
+    const trimmedString = await getTrimmedString({});
+    expect(trimmedString).to.be.eq(INPUT_VAL);
+  });
+
+  it('should remove trailing whitespace', async () => {
+    sandbox.stub(vscode.window, 'showInputBox').resolves(`${INPUT_VAL}  `);
+    const trimmedString = await getTrimmedString({});
+    expect(trimmedString).to.be.eq(INPUT_VAL);
+  });
+
+  it('should remove leading and trailing whitespace', async () => {
+    sandbox.stub(vscode.window, 'showInputBox').resolves(`  ${INPUT_VAL}  `);
+    const trimmedString = await getTrimmedString({});
+    expect(trimmedString).to.be.eq(INPUT_VAL);
+  });
```

Comment:
good good stuff.  Real deal unit tests..awesome.  

Should we consider the the following cases? 
- empty string input
- undefined input 
- null input
- just whitespace input